EnglishFilter/sections/foreign.txt

```diff
@@ -33,6 +33,16 @@ video.gjirafa.com#%#AG_defineProperty('originVideo.requestA.Id', { value: "0" })
 !------------------------------ Arabian ------------------------------
 !---------------------------------------------------------------------
 !
+animesanka.com#%#//scriptlet("prevent-addEventListener", "load", "n")
+!#if (adguard_ext_safari || adguard_app_ios || adguard_ext_android_cb)
+animesanka.com#@##DivTopAd
+animesanka.com#@##tobsideAd
+animesanka.com#@##leftframeAD
+animesanka.com#@##iqadtile5
+animesanka.com#@##ads-sticky
+animesanka.com#@##ad_190x90
+animesanka.com#@##ad-zone-1
+!#endif
```

Comment:
1. Remove your rules.
2. Add domain to the common rules to the section `g207`. `#%#//scriptlet('abort-current-inline-script', 'document.addEventListener', '/google_ad_client/')`.
3. Also add domain to the `content_blockers.txt`. We have a section for `g207`, just `@@||animesanka.com^$generichide`.

Reply:
Haha.. I did everything wrong then.

Reply:
Useless rules. I know something that you do not.

Reply:
What do you mean here https://github.com/AdguardTeam/AdguardFilters/pull/75879#discussion_r583719582?

Reply:
These rules doesn't even work! I tried about 2 hours ago.

Reply:
Which rules and where it doesn't work?

Reply:
These rules doesn't work anymore. Please check, I am tired. https://github.com/AdguardTeam/AdguardFilters/issues/75931

Reply:
This rule `animesanka.com#%#//scriptlet('abort-current-inline-script', 'document.addEventListener', '/google_ad_client/')` works fine on my end.
There is a different domain in #75931, there is `animesanka.club` not `animesanka.com` and adding:
`animesanka.club#%#//scriptlet('abort-current-inline-script', 'document.addEventListener', '/google_ad_client/')`
also works fine on my end.

Reply:
Good catch about the domain, @AdamWr :)

Reply:
@AdamWr I have tried that with asterisk before saying useless no offense. The only working scriptlet on my end is `animesanka.*#%#//scriptlet("prevent-addEventListener", "load", "n")`.